deployments/operator/device/qat/qat.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   namespace: inteldeviceplugins-system
 spec:
   template:
+    metadata:
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/intel-qat-plugin: unconfined
```

Comment:
we don't set this on by default with kustomize but it's provided as a sample overlay. I'm thinking maybe se should keep this here as an example but comment it out.

Reply:
done